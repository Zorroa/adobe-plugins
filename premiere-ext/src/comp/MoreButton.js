import React from 'react'
import { Button } from "react-bootstrap"

function MoreButton(props) {
    
  const callback = props.onpress

  return (
    <Button className="btn btn-dark btn-md" onClick={callback}>
    <svg
      width="1em"
      height="1em"
      viewBox="0 0 18 18"
      className="bi bi-three-dots"
      fill="currentColor"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        fillRule="evenodd"
        d="M3 9.5a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3z"
      />
    </svg>
  </Button>
  )
}

export default MoreButton
