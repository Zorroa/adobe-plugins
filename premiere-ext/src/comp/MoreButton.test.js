import renderer from "react-test-renderer"
import MoreButton from "./MoreButton"

describe("<MoreButton/>", () => {
  it("should render properly", () => {
    const wrapper = renderer.create(
        <MoreButton onpress={()=>{}} />
    )
    expect(wrapper.toJSON()).toMatchSnapshot()
  })
})
