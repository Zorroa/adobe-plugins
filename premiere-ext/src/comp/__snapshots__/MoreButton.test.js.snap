// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoreButton/> should render properly 1`] = `
<button
  className="btn btn-dark btn-md btn btn-primary"
  disabled={false}
  onClick={[Function]}
  type="button"
>
  <svg
    className="bi bi-three-dots"
    fill="currentColor"
    height="1em"
    viewBox="0 0 18 18"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M3 9.5a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3zm5 0a1.5 1.5 0 1 1 0-3 1.5 1.5 0 0 1 0 3z"
      fillRule="evenodd"
    />
  </svg>
</button>
`;
